app/grandchallenge/pages/templates/pages/page_detail.html

```diff
@@ -4,18 +4,9 @@
 {% load bleach %}
 {% load static %}
 
-{% block title %}
```

Comment:
Why was this block deleted?

Reply:
oh, good question! It's because I accidentally deleted everything but content in this file, noticed only later, but then apparently did not put everything back. Fixing this now (also the script block below).